pkg/util/util.go

```diff
@@ -119,6 +118,59 @@ func GetBillingAccountFromSecret(kubeClient kubeclientpkg.Client, namespace, nam
 	return billingAccount, nil
 }
 
+// NewConfigMapCR creates a ConfigMap in specified namespace with default orgParentFolderID value
+func NewConfigMapCR(name, namespace, orgParentFolderID string) *corev1.ConfigMap {
+	return &corev1.ConfigMap{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "ConfigMap",
+			APIVersion: "v1",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      name,
+			Namespace: namespace,
+		},
+		Data: map[string]string{
+			"orgParentFolderID": orgParentFolderID,
+		},
+	}
+}
+
+// ConfigMapExist checks if the configmap is defined before
+func ConfigMapExist(kubeClient kubeclientpkg.Client, name, namespace string) bool {
+	c := &corev1.ConfigMap{}
+	if err := kubeClient.Get(context.TODO(), kubetypes.NamespacedName{Name: name, Namespace: namespace}, c); err != nil {
+		return false
+	}
+
+	return true
+}
+
+// getConfigMap returns a configmap that should contain orgParentFolderID.
```

Comment:
this method has nothing to do with the ParentFolderID, I think it's not necessary to explain that in the comment

Reply:
that part is removed.